src/bootstrap/navs.less

```diff
@@ -28,6 +28,7 @@
 
         > a {
             margin-right: 0;
+            border-bottom: none;
```

Comment:
Why only `border-bottom: none` and not `border: none`, since the active one has no border ?
I've seen also a `margin-bottom: -1px` that could be removed.

Maybe we should clean a little bit more the borders on the tabs :)